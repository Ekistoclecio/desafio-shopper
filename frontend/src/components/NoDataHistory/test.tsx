import { screen } from '@testing-library/react';

import { NoDataHistory } from '.';

import { renderWithTheme } from '@/utils/tests/helpers';

describe('<NoDataHistory />', () => {
  it('should render the NoDataHistory', () => {
    const { container } = renderWithTheme(<NoDataHistory message="teste message" />);

    expect(screen.getByText('teste message')).toBeInTheDocument();

    expect(screen.getByTestId('no-data-image')).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
