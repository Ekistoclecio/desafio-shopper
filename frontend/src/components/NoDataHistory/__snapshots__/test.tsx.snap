// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoDataHistory /> should render the NoDataHistory 1`] = `
<div
  class="MuiBox-root css-1qxi2ho"
>
  <img
    alt="Nenhuma viagem a ser exibida"
    data-testid="no-data-image"
    src="/images/no-data.svg"
    style="width: 360px; max-width: 100%; height: auto;"
  />
  <h4
    class="MuiTypography-root MuiTypography-h4 css-1qv4sbo-MuiTypography-root"
  >
    teste message
  </h4>
</div>
`;
